examples/DatePicker.basic.jsx

```diff
@@ -3,16 +3,45 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { DatePicker } from '@itwin/itwinui-react';
+import { DatePicker, IconButton, Popover } from '@itwin/itwinui-react';
+import { SvgCalendar } from '@itwin/itwinui-icons-react';
 
 export default () => {
   const [currentDate, setCurrentDate] = React.useState(new Date());
+  const [visible, setVisible] = React.useState(false);
   return (
-    <DatePicker
-      date={currentDate}
-      onChange={(date) => {
-        setCurrentDate(date);
-      }}
-    />
+    <Popover
+      content={
+        <DatePicker
+          date={currentDate}
+          onChange={(date) => {
+            setCurrentDate(date);
+            setVisible(false);
+          }}
+          setFocus
+          onBlur={(event) => {
+            if (event.relatedTarget === null) {
+              setVisible(false);
+            }
+          }}
+          onKeyDown={(event) => {
+            if (event.key == 'Escape') {
+              setVisible(false);
+            }
+          }}
```

Comment:
You don't need these extra event handlers, you just need to pass `onVisibleChange={setVisible}` to `Popover` and it should take care of outside clicks and <kbd>Esc</kbd> keypresses

Reply:
Okay, latest commit changes over to that.